applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -403,14 +403,20 @@ config BT_AUDIO_RETRANSMITS
 	  Number of retransmits for the ISO link. 2 retransmits means a total
 	  of 3 packets sent per stream.
 
-config BT_AUDIO_BITRATE
-	int "Bitrate for ISO stream"
+config BT_AUDIO_BITRATE_SINK
+	int "Bitrate for sink ISO stream"
 	default 96000 if BT_AUDIO_BROADCAST_RECOMMENDED || BT_AUDIO_UNICAST_RECOMMENDED
 	default 96000 if BT_AUDIO_UNICAST_CONFIGURABLE || BT_AUDIO_BROADCAST_CONFIGURABLE
 	default 32000 if BT_AUDIO_BROADCAST_16_2_1 || BT_AUDIO_BROADCAST_16_2_2 || BT_AUDIO_UNICAST_16_2_1
 	default 48000 if BT_AUDIO_BROADCAST_24_2_1 || BT_AUDIO_BROADCAST_24_2_2 || BT_AUDIO_UNICAST_24_2_1
 	help
-	  Bitrate for the ISO stream.
+	  Bitrate for the sink ISO stream.
+
+config BT_AUDIO_BITRATE_SOURCE
```

Comment:
Is it then cleaner to use this for all broadcast purposes? Since it is called broadcast_source in the spec?

Reply:
I made a new "BT_AUDIO_BITRATE_BROADCAST_SRC".
I think doing it this way causes the least amount of confusion